rero_ils/modules/circ_policies/api.py

```diff
@@ -70,6 +70,7 @@ class CircPolicy(IlsRecord):
     fetcher = circ_policy_id_fetcher
     provider = CircPolicyProvider
     model_cls = CircPolicyMetadata
+    _schema = 'circ_policies/circ_policy-v0.0.1.json'
```

Comment:
do we need this? specially it is used only by the `tests`
it is already available by 
`current_app.config.get('RECORDS_JSON_SCHEMA')[provider.pid_type]`